atmos_dyamond_summer/README.md

```diff
@@ -0,0 +1,9 @@
+# DYAMOND summer initial conditions
+
+## Overview
+The DYnamics of the Atmospheric general circulation Modeled On Non-hydrostatic Domains (DYAMOND) protocol provides a framework for the intercomparison of global storm-resolving models.
+This artifact contains the initial conditions needed for the DYAMOND summer simulations, which simulate 40 days and nights, begining on 1 August 2016.
+
+## File summmary
+
+- `ifs_oper_T1279_2016080100.nc`: Restart file for DYAMOND summer simulation
```

Comment:
Can you please include the filesize info in the `README.md`

Reply:
Done.